src/pages/settings/Profile/Contacts/NewContactMethodPage.tsx

```diff
@@ -139,6 +145,16 @@ function NewContactMethodPage({loginList, route}: NewContactMethodPageProps) {
                     />
                 )}
             </FormProvider>
+            <ValidateCodeActionModal
+                validatePendingAction={pendingContactAction?.pendingFields?.actionVerified}
+                validateError={validateLoginError}
+                handleSubmitForm={addNewContactMethod}
+                clearError={() => User.clearContactMethodErrors(contactMethod, 'validateLogin')}
```

Comment:
@ntdiary it came from this PR:

https://github.com/Expensify/App/pull/48320

I think Yes because right now on the new Adding Contact Method flow we will need to verify magic code first so we need to save the temporary contact method to submit it later. Is there any issue with it?

Reply:
Oh wait. I think I misunderstood the question. I'll need my laptop to take a look 🥲

Reply:
Yeap that's pretty weird. I think it shouldn't add a new contact method into loginList but let me grab my laptop and see what I find.

Reply:
Yeap that's pretty weird. I think it shouldn't add a new contact method into loginList but let me grab my laptop and see what I find.

Reply:
@ntdiary thanks for raising, this does not seem intended if I understood you well

Reply:
I'm trying to signup with phone number but I can't receive the magic code :( . Can you check the verbose to see where does that contact method come from? (Probably Pusher event).

Reply:
I see. So I think we should check `_isEmpty(validateError)` instead of using `!validateError`. We will fix it in [this PR](https://github.com/Expensify/App/issues/49270)